server/templates/staff/course/course.edit.html

```diff
@@ -33,6 +33,7 @@ <h3 class="box-title">Edit {{ current_course.display_name_with_semester }}</h3>
                     {{ forms.render_field(form.institution, label_visible=true, placeholder='UC Berkeley', type='text') }}
                     {{ forms.render_field(form.website, label_visible=true, placeholder='http://cs61a.org/', type='text') }}
                     <!-- {{ forms.render_field(form.timezone, label_visible=true, type='select') }} -->
+                    {{ forms.render_field(form.autograder_url, label_visible=true, value='https://autograder.cs61a.org/', type='text') }}
```

Comment:
Apologies for nit-picking this, but this will hide the users custom autograder URL. Can you use this please:

`value=current_course.autograder_url or 'https://autograder.cs61a.org/'`